src/huggingface_hub/hf_api.py

```diff
@@ -3597,6 +3600,8 @@ def update_repo_settings(
                 * "auto": The repository is gated, and access requests are automatically approved or denied based on predefined criteria.
                 * "manual": The repository is gated, and access requests require manual approval.
                 * False (default): The repository is not gated, and anyone can access it.
```

Comment:
Should remove `* False (default): The ...` and replace by `* False: The`.

And above in the docstring: 

```
The gated status for the repository. If set to `None` (default), the `gated` setting of the repository won't be updated
```